server/src/infra/repositories/asset.repository.ts

```diff
@@ -598,7 +598,7 @@ export class AssetRepository implements IAssetRepository {
       .select(`COUNT(asset.id)::int`, 'count')
       .addSelect(truncated, 'timeBucket')
       .groupBy(truncated)
-      .orderBy(truncated, 'DESC')
+      .orderBy(truncated, options.order === AssetOrder.ASC ? 'ASC' : 'DESC')
```

Comment:
Can't we just do `options.order.toUpperCase()` instead?

Reply:
I don't know about that. I don't like the idea of mixing SQL syntax and enum values

Reply:
Hm ok that's fair. Coupling those would result in the query breaking in case we were to ever rename the enum value. That's actually bad, true!

Reply:
As soon as we add new values to this enum, we'll have to update that part too and I don't know if typescript / typeorm can alert it. It will definitely be a disaster :laughing: